src/bundle/Resources/public/scss/_backdrop.scss

```diff
@@ -0,0 +1,9 @@
+.ibexa-backdrop {
+    position: fixed;
+    z-index: 100;
```

Comment:
I have checked it in the interface as well as our scss files and `z-index: 100` seems to be OK. But in the future, it would be good to have some guidelines etc. 

Interestingly, there is `ez-backdrop` in Page Builder scss and it also has `z-index: 100`:
https://github.com/ezsystems/ezplatform-page-builder/blob/7ae5586359817c37807d9701adcb474d755013c1/src/bundle/Resources/public/scss/_page-info-bar.scss#L271